src/helpers/Logger.js

```diff
@@ -0,0 +1,43 @@
+const winston = require('winston');
+// Supress error handling
+winston.emitErrs = true;
+
+const logger = new winston.Logger({
+  transports: [
+    new (winston.transports.Console)({
+      timestamp: () => {
+        return new Date().toISOString();
+      },
+      formatter: (options) => {
+        const result = {
+          timestamp: options.timestamp(),
+          level: options.level.toUpperCase()
```

Comment:
Might be need at some point to add matching to our agreed-upon log levels (https://github.com/NYPL/engineering-general/issues/6). I noticed Winston uses slightly different levels (e.g. `WARN` vs `WARNING`): https://github.com/winstonjs/winston#logging-levels. I guess we haven't finalized the standard yet.